internal/prompts/app_select_test.go

```diff
@@ -1789,7 +941,7 @@ func TestPrompt_AppSelectPrompt_FlatAppSelectPrompt(t *testing.T) {
 		expectedStdout             string
 		expectedStderr             string
 	}{
-		"selects a saved applications using prompts": {
+		"returns a saved applications using prompts": {
```

Comment:
📚 Wording updates to start all of these cases with either "returns" or "errors" follow this comment.